source/gui/addonGui.py

```diff
@@ -332,14 +344,9 @@ def installAddon(self, addonPath, closeAfter=False):
 			if not addonVersionCheck.hasAddonGotRequiredSupport(bundle):

```

Comment:
I've noticed in Alpha builds that these new custom dialogs sometimes do not get the foreground. This can probably be fixed by calling gui.mainFrame.prePopup just before this line. We also need a call of postPopup somewhere probably, but I"m very unfamiliar with this part of the code base.